### PR TITLE
Execa Logs Streaming

### PR DESCRIPTION
**Pull request by ralikio**
**Description**

Changes proposed in this pull request:

- Improved logging of `execa` - before the change logs were printed only when a command finished. Because of that in the case of timeouts the logs were not visible.
- Prompt confirmation - because of introduce confirmation prompt in kcp, the tests require sending `Y` to invoked subprocesses.
- Switch params for the above.

**Related issue(s)**

### CODE REVIEW

File: tests/fast-integration/skr-aws-upgrade-integration/index.js [hunk 1]
```diff
@@ -44,6 +44,11 @@ describe('SKR-Upgrade-test', function() {
     return options;
   };
 
+  it(`Perform kcp login`, async function() {
+    console.log('Performing kcp login');
+    await kcp.login();
```
Comment (PK85) [inline tests/fast-integration/skr-aws-upgrade-integration/index.js:49]: how it will help? kcp.login() is called in the getOrProvisionSKR function

Comment (ralikio) [reply tests/fast-integration/skr-aws-upgrade-integration/index.js:49]: You are right.

**Comment by ralikio** [comment]
/test 
pre-main-kyma-gardener-gcp-eventing-upgrade

**Pull request merged**